packages/nuxt/src/module.ts

```diff
@@ -16,34 +14,25 @@ export default defineNuxtModule<ModuleOptions>({
   },
   defaults: {},
   setup(_moduleOptions, nuxt) {
-    const resolver: Resolver = createResolver(import.meta.url);
-
-    const pathToClientInit = findDefaultSdkInitFile('client');
-
-    if (pathToClientInit) {
-      nuxt.hook('app:templates', nuxtApp => {
-        if (nuxtApp.rootComponent) {
-          try {
-            addImportStatement(nuxtApp.rootComponent, buildSdkInitFileImportSnippet(pathToClientInit));
-          } catch (err) {
-            // eslint-disable-next-line no-console
-            console.error(`[Sentry] Could not add import statement to root component. ${err}`);
-          }
-        }
-      });
+    const moduleDirResolver = createResolver(import.meta.url);
+    const buildDirResolver = createResolver(nuxt.options.buildDir);
+
+    const clientConfigFile = findDefaultSdkInitFile('client');
+
+    if (clientConfigFile) {
+      nuxt.options.alias['#sentry-client-config'] = buildDirResolver.resolve(`/${clientConfigFile}`);
+
+      injectSentryConfigImport('#sentry-client-config');
+
+      addPlugin(moduleDirResolver.resolve('./runtime/plugins/sentry.client'));
     }
 
-    if (resolver) {
-      addPlugin(resolver.resolve('./runtime/plugins/sentry.client'));
+    const serverConfigFile = findDefaultSdkInitFile('server');
+
+    if (serverConfigFile) {
+      nuxt.options.alias['#sentry-server-config'] = buildDirResolver.resolve(`/${serverConfigFile}`);
```

Comment:
l: Maybe there's a way to get rid of this alias indirection by rendering `buildDirResolver.resolve(/${clientConfigFile})` directly into the template.